src/njoy21/input/MATXSR/Card10/Matgg/test/Matgg.test.cpp

```diff
@@ -0,0 +1,40 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "MATXSR Card10 Matgg",
+          "[MATXSR], [Card10], [Matgg]" ){
+  GIVEN( "valid inputs" ){
+    WHEN( "valid values are provided" ){
+      for( int i : { 1, 10, 100, 1000, 10000 } ){
+        iRecordStream<char> iss( std::istringstream( std::to_string( i ) ) );
+
+        THEN( "the value can be verified" ){
+          REQUIRE( i == argument::extract< MATXSR::Card10::Matgg >( iss, 1 ).value );
+        }
+      }
+    }
+
+    WHEN( "no value is provided" ){
+      iRecordStream<char> iss( std::istringstream( " /" ) );
+
+      THEN( "the default is returned based on the matno value" ){
+        REQUIRE( 1000 == argument::extract< MATXSR::Card10::Matgg >( iss, 1000 ).value );
```

Comment:
So, this one actually confused me.  I couldn't find any good examples to clarify this.  If the last two digits are not '00', do they end up as '00' or whatever they were to begin with?

Reply:
Take for example U-235 which has an ENDF Material number of `9228`. However, all photon interactions are done on an elemental rather than an isotopic basis. The way you get the elemental Material ID is to divide by 100 and then multiply by 100. When done as integers, this truncates the value and when you multiply by 100, you just get the elemental Material number of `9200`.

Reply:
As I recall, there are some older material number representations that are <100.  How does that work in this scheme?

Reply:
Material numbers <100 are still current, but they are generally the thermal scattering files for which we wouldn't run MATXSR.

Reply:
Alright, so in the first test situation I loop over values and have the matno value set to 1 every time.  Is that right? Is this value (matgg) just meant to override the default value, such that it has no relation?